todo_app/data/trello_items.py

```diff
@@ -5,76 +5,65 @@
 load_dotenv()
```

Comment:
We _should_ find we don't need to explicitly load the `.env` file here - Flask does it for us automatically.